python/felis/tap.py

```diff
@@ -413,6 +414,24 @@ def visit_column(self, column_obj: datamodel.Column, table_obj: Table) -> Tap11B
             arraysize = column_obj.votable_arraysize or "*"
         column.arraysize = arraysize
 
+        def _is_int(s: str) -> bool:
+            try:
+                int(s)
+                return True
+            except ValueError:
+                return False
+
+        # Handle the deprecated size attribute
+        if arraysize is not None and arraysize != "":
+            if isinstance(arraysize, int):
+                column.size = arraysize
+            elif _is_int(arraysize):
+                column.size = int(arraysize)
+            elif arraysize != "*" and bool(re.match(r"^[0-9]*\*$", arraysize)):
```

Comment:
Should be `r"^[0-9]+\*$"`

Reply:
That suggests to me that there was some test coverage missing.

Reply:
(Which now that I look at the changed files the PR is indeed telling me exactly that...)

Reply:
The `tap` module does not have good coverage at all, including for this change. Fixing that is probably beyond the scope of this PR.